declarations/Facebook.json

```diff
@@ -4,7 +4,7 @@
     "Terms of Service": {
       "fetch": "https://www.facebook.com/legal/terms/plain_text_terms",
       "select": [
-        ".ad2k81qe"
+        "div[role=main]"
       ],
       "remove": [
         ".mdeji52x"
```

Comment:
An improvement i missed last time, you could cleaning the version by adding the following remove selector:
`.bdao358l.om3e55n1.g4tp4svg.alzwoclg.cqf1kptm.jez8cy9q.gvxzyvdx.i85zmo3j.q6feio67.srn514ro`